### PR TITLE
rg-202: Create Popup Hint For Password

### PR DESCRIPTION
**Pull request by RevolveStrm**
image
image
image

### CODE REVIEW

**Comment by RevolveStrm** [comment]
[Create the pop up hint for password instead long validation error message](https://trello.com/c/7V7AoUeG/202-create-the-pop-up-hint-for-password-instead-long-validation-error-message)

**Review by ValentinaTenus — APPROVED** [review]

**Review by Valentyn13 — APPROVED** [review]

**Comment by siefimov** [comment]
Hint has effected the icon:
image

**Review by siefimov — CHANGES_REQUESTED** [review]
HInt has effected the icon:
image

**Review by jurok3x — APPROVED** [review]

**Review by honkster-nazarii — APPROVED** [review]

**Review by siefimov — APPROVED** [review]

File: frontend/src/bundles/common/components/hint/hint.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import clsx from 'clsx';
+import { type FC } from 'react';
+import { Tooltip as ReactTooltip } from 'react-tooltip';
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/common/components/hint/hint.tsx:3]: Don't use components straight from library.
We already have Tooltip component for it

File: frontend/src/bundles/common/components/hint/hint.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                className={styles.hint__anchor_container}
+            />
+            <ReactTooltip
+                anchorSelect={`.${styles.hint__anchor_container}`}
+                className={styles.hint__popup}
+            >
+                {rows?.length &&
+                    rows.map((row: HintRow) => (
+                        <div key={row.text} className={styles.hint__row}>
+                            <img
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/common/components/hint/hint.tsx:32]: There is Icon component

**Review by Vitaliy-Kalenichenko — CHANGES_REQUESTED** [review]

**Comment by RevolveStrm** [comment]
image

File: frontend/src/bundles/common/components/hint/hint.tsx (hunk 2 above)
Comment (RevolveStrm) [reply frontend/src/bundles/common/components/hint/hint.tsx]: I changed code with an Icon component

File: frontend/src/bundles/common/components/hint/hint.tsx (hunk 1 above)
Comment (RevolveStrm) [reply frontend/src/bundles/common/components/hint/hint.tsx:3]: But we can't pass any components in Tooltip as a property, only a text as a string, that's why i created a Hint component. So i still need to use Tooltip and modify it?

**Review by MaxPoplavskiy — APPROVED** [review]

**Review by Icerock10 — CHANGES_REQUESTED** [review]
Conflicts

File: frontend/src/bundles/common/components/hint/hint.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import clsx from 'clsx';
+import { type FC } from 'react';
+import { Tooltip as ReactTooltip } from 'react-tooltip';
```
Comment (Icerock10) [inline frontend/src/bundles/common/components/hint/hint.tsx:3]: You can adapt current `<Tooltip /> `component for your needs

**Review by Icerock10 — CHANGES_REQUESTED** [review]

**Comment by RevolveStrm** [comment]
After changes: 
image

**Review by Icerock10 — CHANGES_REQUESTED** [review]
Conflicts

**Review by Icerock10 — APPROVED** [review]

**Pull request merged**